source/opt/basic_block.h

```diff
@@ -64,6 +64,11 @@ class BasicBlock {
     return const_iterator(&insts_, insts_.cend());
   }
 
+  iterator tail() {
```

Comment:
I prefer "back()" as more idiomatic. That's what standard containers have (vector, queue, deque, array, string, list...)